string id
---
cavity_detection_msgs/Roi roi
